### DISCUSSION TITLE
Remove mobile app from 2FA

### DISCUSSION THREAD

**Discussion by cuducos** [community/community#23098] [New to GitHub]
<p>Hi,</p>
<p>I deleted GitHub’s mobile app from my phone and I am not using it anymore. However, I could not find how to remove the mobile app as a two-factor authentication (and have, for example, one-time password generated via another app as default).</p>
<ul>
<li>In <a href="https://github.com/settings/security" class="inline-onebox" rel="noopener nofollow ugc">Sign in to GitHub · GitHub</a> I can see the mobile app, but there’s no button to remove it</li>
<li>In <a href="https://github.com/settings/two_factor_authentication/configure" rel="noopener nofollow ugc">https://github.com/settings/two_factor_authentication/configure</a> again I can see the mobile app is active, but there’s no button to deactivate it</li>
</ul>
<p>Does anyone know how can I remove GiHub mobile app as a 2FA?</p>
<p>Many thanks,</p>
<p>Cuducos</p>

**Comment by jfullstackdev** [comment]
<p>I’m not very sure of this, but how about revoking that at your Settings on Web UI?</p>
<p>check these two links,</p>
<p><a href="https://github.com/settings/apps">https://github.com/settings/apps</a></p>
<p><a href="https://github.com/settings/developers">https://github.com/settings/developers</a></p>

**Comment by cuducos** [comment]
<aside class="quote no-group" data-username="jdevstatic" data-post="2" data-topic="226985">
<div class="title">
<div class="quote-controls"></div>
 jdevstatic:</div>
<blockquote>
<p>how about revoking that at your Settings on Web UI?</p>
</blockquote>
</aside>
<p>The GitHub mobile app does not show up in any of these UIs : (</p>

**Comment by ottobar** [comment, marked as answer]
<p>I had this same problem and here is what I did to solve:</p>
<ul>
<li>Re-install mobile app</li>
<li>When I reinstalled, I was still signed into GitHub on the mobile app</li>
<li>Sign out of GitHub in the mobile app</li>
<li>Uninstall the mobile app</li>
<li>:tada:</li>
</ul>

**Comment by cuducos** [comment]
<p>Wow! Many thanks, <a class="mention" href="/ottobar">@ottobar</a> — that worked : )</p>

**Comment by Defeatomizer** [comment]
<p>Does this remove the device from GitHub Mobile? I tried following these instructions and my device didn’t get unregistered (instead, it got registered once more).</p>

**Comment by samuel-lucas6** [comment]
<p>I have the same issue as <a class="mention" href="/defeatomizer">@Defeatomizer</a>, so this solution doesn’t seem to work in all cases. I unfortunately uninstalled without signing out originally. Turning off 2FA and re-enabling it doesn’t even fix this problem. It defaults to asking for GitHub Mobile now every time I log in, even after ticking the checkbox.</p>

**Comment by frumpled** [comment]
<p>Any word on this?<br>
I’m also having the same issue <a class="mention" href="/samuel-lucas6">@samuel-lucas6</a> mentioned.</p>

**Comment by nicfitzgerald** [comment]
<p>Just want to throw in that - as of 6/27, I am also having this issue.</p>
<p>On my first round, I logged in, but uninstalled the app and revoked app access in my account settings. This second time, I redownloaded the app, got a revocation error, logged <em>back</em> in and now I see two devices registered under Github Mobile :frowning:</p>

**Comment by davesgonechina** [comment]
Answer doesn't work for me either, using Android app.
